recipes/gtest/all/conandata.yml

```diff
@@ -5,6 +5,9 @@ sources:
   "1.10.0":
     url: "https://github.com/google/googletest/archive/release-1.10.0.tar.gz"
     sha256: "9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb"
+  "cci-20210126":
```

Comment:
Sorry that I did not notice it the first time. But it has to be a dot instead of a dash. See also https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

`cci.20210126`

Reply:
No worries, I should read better :).